app/Http/Controllers/ApiPagesController.php

```diff
@@ -2,15 +2,84 @@
 
 namespace Suyabay\Http\Controllers;
 
+use Auth;
+use Suyabay\AppInfo;
+use Firebase\JWT\JWT;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Illuminate\Database\QueryException;
+use Suyabay\Http\Controllers\Controller;
+
 class ApiPagesController extends Controller
 {
+    /**
+     * Displays the index page of the API.
+     *
+     * @return \Illuminate\Http\Response
+     */
     public function index()
     {
         return view('api.pages.index');
     }
 
+    /**
+     * Display list of apps the user have created and a link to create new appp.
+     *
+     * @return \Illuminate\Http\Response
+     */
     public function myApp()
     {
         return view('api.pages.myapp');
     }
+    
+    /**
+     * Generate a token for user with passed Id.
+     *
+     * @param int $userId
+     *
+     * @return string
+     */
+    public function generateToken()
+    {
+        $appSecret = getenv('APP_SECRET');
+        $jwtAlgorithm = getenv('JWT_ALGORITHM');
+        $timeIssued = time();
+        $tokenId = base64_encode(getenv('TOKENID'));
+        $token = [
+            'iss'  => 'http://suyabay-staging.herokuapp.com/',
```

Comment:
**Code Quality - Nitpick**

Hard coding the issuer to this particluar url is not effective. The site url can change at anytime, thus meaning the developer has to come to the code to change it. Please use a laravel method to get the domain url and use here @andela-araimi